### PR TITLE
feat: Add new metric kube_pod_scheduler

### LINKED ISSUE #2207: schedulerName in kube_pod_* metrics group
**What would you like to be added**:
It would be great if we will have schedulerName in exported kube_info_metrics.
**Why is this needed**:
When we have more than one scheduler in the cluster, we need to aggregate some data by scheduler.
**Describe the solution you'd like**
It could be a metric called kube_pod_scheduler_name with labels set {pod, namespace, uid, schedulerName (or just name)}. Value can be taken from .spec.schedulerName.

### PR DESCRIPTION
**Pull request by adinhodovic**
**What this PR does / why we need it**:

Adds a new pod metric `kube_pod_scheduler_name` that allows seeing which scheduler scheduled the pod. Allows aggregation by scheduler.

**How does this change affect the cardinality of KSM**: *(increases, decreases or does not change cardinality)*

Increases.

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #2207

### CODE REVIEW

File: internal/store/pod.go [hunk 1]
```diff
@@ -1708,6 +1709,27 @@ func createPodServiceAccountFamilyGenerator() generator.FamilyGenerator {
...
+func createPodSchedulerNameFamilyGenerator() generator.FamilyGenerator {
+	return *generator.NewFamilyGeneratorWithStability(
+		"kube_pod_scheduler_name",
+		"The scheduler name for a pod.",
+		metric.Gauge,
+		basemetrics.ALPHA,
+		"",
+		wrapPodFunc(func(p *v1.Pod) *metric.Family {
+			m := metric.Metric{
+				LabelKeys:   []string{"scheduler_name"},
```
Comment (dgrisonnet) [inline internal/store/pod.go:1721]: I am wondering if having something like `kube_pod_scheduler{name="foo"}` could be better to remove the redundancy with the metric name its label.

Comment (adinhodovic) [reply internal/store/pod.go]: Think that makes sense.  Updated.

**Review by dgrisonnet — APPROVED** [review]
/lgtm
/approve

**Pull request merged**